test/logoutrequest_test.rb

```diff
@@ -91,18 +81,24 @@ class RequestTest < Minitest::Test
     end
 
     describe "when the settings indicate to sign (embedded) logout request" do
-      it "created a signed logout request" do
-        settings = OneLogin::RubySaml::Settings.new
-        settings.idp_slo_target_url = "http://example.com?field=value"
-        settings.name_identifier_value = "f00f00"
+
+      let(:settings2) { OneLogin::RubySaml::Settings.new }
```

Comment:
Line length is much less of an issue than non-descriptive names. I can scan to the right, or wrap to the next line, if things get long, As this stands, I need to deal with multiple files to understand the full context of this test.